### ISSUE TITLE
What should I pay attention to when collecting data for fine-tuning?

### ISSUE THREAD

**Issue by qc-1** [physical-superintelligence-lab/Psi0#31]
Thank you for your wonderful work
I have a question: What should I pay attention to when collecting data for fine-tuning? We remotely operated G1 to collect about 40 pieces of data, each lasting about 5 seconds. The task was to move the object from the table to the nearby chair, but the deployment effect was very poor. The robot didn't even pick up the object. May I ask why this is happening
Thanks for your reply.

**Comment by songlin** [comment]
Hi @qc-1,

The task "move the object from the table to the nearby chair" can be tricky, depending on how far the distance is. We observed that AMO can not walk in a straight line without tuning the speed vx/vy.  However, the first part of the task, "picking up the object", should be easy. You should expect picking up the object to be very robust, provided the data quality is fine. 

I suggest you first try to replay the recorded data on the same G1. In general, the replay should have a high success rate, at least it should closely replicate the teleoperated motion.

Once the data is validated through replaying, the next step is to observe the training loss during fine-tuning.  Check if the L1 loss is small enough.

Best,
Songlin

**Comment by songlin** [comment]
Closing the issue for now, feel free to re-open it once you get updates.
